### PR TITLE
Add lander

### PR DESCRIPTION
**Pull request by craig-mulligan**
WIP.

### CODE REVIEW

**Comment by jviotti** [comment]
Woohoo 🎉 This is VERY exciting! I'll give this a go and a closer look next week :)

File: www/js/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+'use strict';
+const _ = require('lodash');
+const platform = require('platform');
+const $ = require('jquery');
+const downloads = $('#downloads').data('downloads');
+
+// Find and download link based on detected OS
+// Defualt is first link the matching is unsuccessful
+const dynamicDownload = _.find(downloads, (d) => {
+  new RegExp(d.OS).test(platform.os.family);
```
Comment (stefan-mihaila) [inline www/js/index.js]: Shouldn't we also take architecture into consideration?

Comment (lurch) [inline www/js/index.js]: See also #876

**Comment by jviotti** [comment]
This looks really nice, however we need to put a lot of effort in reducing the amount of configuration needed.

 @craig-mulligan do you mind explaining how the current implementation works? As far as I can see, you're using Webpack, and the project maintainer needs to do a build step (`landr deploy`) which builds the website and puts it into `gh-pages`, right?

If so, any reason why you ditched Jekyll templates instead, which wouldn't require the build step?

**Comment by alexandrosm** [comment]
cc @pimterry

-- 

*Alexandros Marinos*

Founder & CEO, Resin.io


+1 206-637-5498

@alexandrosm

**Comment by craig-mulligan** [comment]
@jviotti true, there is alot of configuration. I hope to hide that and then only show it if you run smthing like `landr eject` or add the possibility of merging in only things you change (still experimenting) 

Yes, `landr deploy` will build everything + push to gh-pages. 

I moved away from jekyll templates because it means we'd have to commit the templates/js/scss to the users repo, this would mean updates to landr wouldn't update any of that source (like it does now).

I started off thinking webpack could load files using webpack and then spit out a directory like we have here: https://github.com/craig-mulligan/jek-test but then as I mentioned where do you edit the templates?  In the directory we just created or by passing landr some new templates? It essentially creates a double build step which I thought was confusing. But very open to idea's that could make it less so.

**Comment by jviotti** [comment]
Based on today's landr meeting, I'll close this one for now. We'll adopt landr in landr first (which will likely change various things), and then adopt it in Etcher.

**Pull request closed without merging**